platforms/ide/tooling-api/src/testFixtures/groovy/org/gradle/integtests/tooling/fixture/GradleVersionSpec.java

```diff
@@ -103,4 +126,15 @@ public boolean isSatisfiedBy(GradleVersion element) {
         return Specs.intersect(specs);
     }
 
+    private void validateLowestTestedVersion(String constraint, String value, GradleVersion minVersion) {
```

Comment:
I'd add some explanation why we do the validation.

```suggestion
    private void validateLowestTestedVersion(String constraint, String value, GradleVersion minVersion) {
    // The cross-version tests should fail if the specified version range is outside of the supported set of versions
```